spec/models/person_spec.rb

```diff
@@ -27,4 +27,20 @@
       expect(subject.gender).to eq(subject.gender.downcase)
     end
   end
+
+  describe 'admin?' do
```

Comment:
Thanks but we don't need it anymore haha! With enum there is a built in rails method that we don't need to test.